webflow-hacks/form-hacks/17/17.js

```diff
@@ -20,7 +20,7 @@ document.addEventListener('DOMContentLoaded', () => {
   if (!formButtonDown) return;
   // increase counter input value upon clicking on up button
   formButtonUp.addEventListener('click', () => {
-    // convert input value from a number to a string
+    // convert input value from a number to a string and assign it to constant stringValue
     const stringValue = JSON.stringify(++x);
```

Comment:
I think you can just use `parseInt` in Javascript.

JSON.strigify is to convert an object to the JSON equivalent

Reply:
you mean toString()?

Reply:
Sorry for the mixup, I meant `toString`. Thanks.